Diagnostics/HealthChecker/Analyzer/Invoke-AnalyzerFrequentConfigurationIssues.ps1

```diff
@@ -372,6 +373,106 @@ function Invoke-AnalyzerFrequentConfigurationIssues {
             }
             Add-AnalyzedResultInformation @params
         }
+
+        # TODO: Move this check to the new IIS section that we'll add to HC in near future - See issue: 1363
+        if (($iisWebSitesWithHstsSettings.Hsts.NativeHstsSettings.enabled -notcontains $true) -and
+            ($iisWebSitesWithHstsSettings.Hsts.HstsViaCustomHeader.enabled -notcontains $true)) {
+            $params = $baseParams + @{
+                Name    = "HSTS Enabled"
+                Details = $false
+            }
+            Add-AnalyzedResultInformation @params
+        } else {
+            $showAdditionalHstsInformation = $false
+            foreach ($webSite in $iisWebSitesWithHstsSettings) {
+                $hstsConfiguration = $null
+                $isExchangeBackEnd = $webSite.Name -eq "Exchange Back End"
+                $hstsMaxAgeWriteType = "Green"
+
+                if (($webSite.Hsts.NativeHstsSettings.enabled) -or
+                    ($webSite.Hsts.HstsViaCustomHeader.enabled)) {
+                    $params = $baseParams + @{
+                        Details          = "HSTS Enabled: $($webSite.Name)"
+                        DisplayWriteType = if ($isExchangeBackEnd) { "Red" } else { "Green" }
+                    }
+                    Add-AnalyzedResultInformation @params
+
+                    if ($isExchangeBackEnd) {
+                        $showAdditionalHstsInformation = $true
+                        $params = $baseParams + @{
+                            Details                = "HSTS on 'Exchange Back End' is not supported and can cause issues"
+                            DisplayWriteType       = "Red"
+                            DisplayCustomTabNumber = 2
+                        }
+                        Add-AnalyzedResultInformation @params
+                    }
+
+                    if (($webSite.Hsts.NativeHstsSettings.enabled) -and
+                    ($webSite.Hsts.HstsViaCustomHeader.enabled)) {
+                        $showAdditionalHstsInformation = $true
+                        Write-Verbose "HSTS conflict detected"
+                        $params = $baseParams + @{
+                            Details                = "HSTS configured via customHeader and native IIS control - please remove one configuration"
+                            DisplayWriteType       = "Red"
+                            DisplayCustomTabNumber = 2
+                        }
+                        Add-AnalyzedResultInformation @params
```

Comment:
When we are providing that this is bad. We just point to the doc which doesn't call out that you can't do both, or at least as far as I can tell. Should we update the doc, explain why this is bad in the script or in a CSS-Exchange Doc?

Reply:
Need to understand how this behaves if both ways are configured (will test that later today). We can then decide if we want to have this covered in the HSTS documentation or if it makes sense to maintain a separate check documentation.

Reply:
Okay, I've tested this one. If you configure HSTS via native HSTS control + customHeader, the setting configured via native will be used. If you remove the native HSTS configuration, IIS will send the customHeader configuration. 
I'd suggest changing this from DisplayWriteType Red to Yellow as it should not break anything. Do you agree?

Reply:
Addressed with the latest commit.